pre-commit/run-go-fmt-import.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+LIST_OF_FILES=$(goimports -l -w "$@")
+# print a list of affected files if any
+echo "$LIST_OF_FILES"
+if [ -n "$LIST_OF_FILES" ]; then
+    exit 1
```

Comment:
for my understanding, why does this need to capture and echo output to stdout, then exit 1 if there is no output, as opposed to `goimports -l -w "$@" 2>&1` or just `goimports -l -w "$@"`.

Reply:
since this started off from someone else's work I wanted to minimize changes to it. I treated it as a black box that 1. seems to work and 2. doesn't do anything nefarious 3. does not make network calls.

maybe they were doing something more than `exit 1`? consistency with the other hooks they wrote?

Reply:
It's unclear to me how we're doing those attributions I'm asking about it in slack there are some inconsistencies https://hpe-aiatscale.slack.com/archives/CRYLQP7MW/p1687372579874599
my plan is to link to the repo in the commit description.

Reply:
sgtm, thanks